capellambse/model/__init__.py

```diff
@@ -573,6 +573,7 @@ def update_diagram_cache(
             format=image_format,
             index=create_index,
             force=force,
+            background=None,
```

Comment:
This would also make for a nice (optional) method argument, which simply defaults to the current behavior, i.e. "False".